flake.nix

```diff
@@ -28,6 +28,8 @@
         nixUnstable
         # Dynamically load nix envs
         nix-direnv
+        # Add git client to shell, it reads host configuration
+        git
```

Comment:
Wasn't this already part of the other PR?

Reply:
Ah it was, I didnt had access so I made a tree of subsequent branches. Will merge first the addition of git itself and then pull on those and merge them back to make it clean :+1: